package.json

```diff
@@ -66,7 +66,7 @@
     "@typescript-eslint/eslint-plugin": "5.9.1",
     "@typescript-eslint/parser": "5.9.1",
     "acorn-globals": "6.0.0",
-    "ajv": "8.6.3",
+    "ajv": "8.10.0",
```

Comment:
Is there a reason this needs to be updated for this PR rather than just letting renovate handle the upgrade?

Reply:
Some ajv options became available in a newer version, like `source`. 

(I'm also unsure why Renovate has not yet handled this dependency, the used version is quite out of date.)